commands/moon.js

```diff
@@ -0,0 +1,16 @@
+const embeds = require("../embeds.js");
+var shibesSent = 0;
+
+function goToTheMoon(message) {
+    shibesSent = shibesSent + 1;
+    message.channel.send(embeds.successEmbed(`A shibe has been sent to the moon! ${shibesSent} shibes has been sent to the moon. much wow`))
+    console.log('doge has been sent to the moon, much wow.')
+}
+ module.exports = {
+	name: 'moon',
+	description: 'Send a shibe to the moon!',
+	dcPermissions: ['EMBED_LINKS'],
+	execute(message, args) {
+		goToTheMoon()
+	},
+};
```

Comment:
```suggestion
const embeds = require("../embeds.js");
var shibesSent = 0;

module.exports = {
    name: 'moon',
    description: 'Send a shibe to the moon!',
    dcPermissions: ['EMBED_LINKS'],
    execute(message, args) {
      shibesSent = shibesSent + 1;
      message.channel.send(embeds.successEmbed(`A shibe has been sent to the moon! ${shibesSent} shibe(s) has been sent to the moon. much wow`))
      console.log('doge has been sent to the moon, much wow.')
    },
};
```

cleanup as talked about over discord DMs